.github/workflows/go.yml

```diff
@@ -155,8 +155,8 @@ jobs:
     - name: Check tidy
       run: make test-tidy
 
-  verify-spelling:
-    name: Verify common spelling mistakes
+  verify:
```

Comment:
wondering verify-docs or verify-mds a better name? 
This may not work if spelling verification can do checks for comments in the code. Is that correct?

Reply:
yes.. spell check also looks at code comments for common mistakes